code/game/objects/structures/window.dm

```diff
@@ -437,6 +438,7 @@
 	icon_state = "clockwork_window_single"
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	max_integrity = 100
+	armor = list(melee = 60, bullet = 25, laser = 0, energy = 0, bomb = 25, bio = 100, rad = 100, fire = 80, acid = 100)
```

Comment:
Like, maybe just have this and nix the fireaxe change. Or, make the fireaxe change much more visible in both the PR desc and changelog, with verbiage such as "this was done partly by halving fireaxe damage to windows".